### DISCUSSION TITLE
posts/webpack/webpack5_base

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#407] [Announcements]
# posts/webpack/webpack5_base

webpack5_base webpack5参考文档地址 oneOf 每个文件只被一个loader配置处理 include, exclude 包含和排除文文件,只在js loader中配置 babel和 eslint,一般不处理css等 exclude: /(node_modules)/, // 排除node_modules, // include :resolve(__dirname,'../src'),

http://localhost:8080/posts/webpack/webpack5_base.html

**Comment by anran919** [comment]
hello
